deployment/migrate-tas.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Migrate is for existing installations of versions prior to 4.0.0, does it make sense for TAS?

Reply:
No - we took it out of the README but didn't stop the script from being generated, will do that now